lib/findbugs_translate_checkstyle_format/translate.rb

```diff
@@ -22,7 +22,13 @@ def trans(xml)
 
       bug_instances = [bug_instances] if bug_instances.is_a?(Hash)
       bug_instances.each do |bug_instance|
-        source_lines = bug_instance['SourceLine']
+        source_lines = if bug_instance.has_key?('SourceLine')
```

Comment:
__[rubocop]__
* Use `Hash#key?` instead of `Hash#has_key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)

<!-- Comment by [SideCI](https://sideci.com) -->